examples/data-transfer-pg/main.py

```diff
@@ -0,0 +1,91 @@
+''' Main module. '''
+import sys
+import os
+import time
+import pyodbc
+
+CONNECTION_STRING = 'DRIVER={{PostgreSQL Unicode}};SERVER={server};DATABASE={database};UID={username};PWD={password};'
+
+
+def main():
+    ''' App entrypoint. '''
+    # Wait for postgres database server to fully spawn.
+    time.sleep(5)
+
+    db1_name = os.environ['DB1_NAME']
+    db1_conn = get_connection(
+            os.environ['DB1_HOST'], 
+            db1_name, 
+            os.environ['DB1_USERNAME'], 
+            os.environ['DB1_PASSWORD']
+        )
+    db1_cur = db1_conn.cursor()
+
+    db2_name = os.environ['DB2_NAME']
+    db2_conn = get_connection(
+            os.environ['DB2_HOST'], 
+            db2_name,
+            os.environ['DB2_USERNAME'], 
+            os.environ['DB2_PASSWORD']
+        )
+    db2_cur = db2_conn.cursor()
```

Comment:
You can move this boilerplate code into a separate function and only get what you need from it. Like this:
```py
db1_conn, db2_conn = connect_to_databases()
```